js/main.js

```diff
@@ -28,9 +32,53 @@ $entryForm.addEventListener('submit', function (event) {
     entryNotes: $entryNotes.value,
     entryId: data.nextEntryId++
   };
-  data.entries.unshift(entryFormObject);
+  data.entries.push(entryFormObject);
   $entryImage.setAttribute('src', 'images/placeholder-image-square.jpg');
   $entryImage.setAttribute('alt', 'Placeholder Image');
   $entryImage.setAttribute('title', 'Placeholder Image');
   $entryForm.firstElementChild.reset();
+  setEntry(entryFormObject);
+});
+
+function setEntry(dataEntry) {
+  var $newEntry = document.createElement('li');
+  $newEntry.className = 'row';
+  $newEntry.setAttribute('id', dataEntry.entryId);
+  var $leftColumn = document.createElement('div');
+  $leftColumn.className = 'column-half left';
+  $newEntry.appendChild($leftColumn);
+  var $leftImage = document.createElement('img');
+  $leftImage.setAttribute('src', dataEntry.entryUrl);
+  $leftColumn.appendChild($leftImage);
+  var $rightColumn = document.createElement('div');
+  $rightColumn.className = 'column-half right';
+  $newEntry.appendChild($rightColumn);
+  var $rightTitle = document.createElement('h2');
+  $rightTitle.textContent = dataEntry.entryTitle;
+  $rightColumn.appendChild($rightTitle);
+  var $rightNotes = document.createElement('p');
+  $rightNotes.textContent = dataEntry.entryNotes;
+  $rightColumn.appendChild($rightNotes);
+  $ulEntries.prepend($newEntry);
```

Comment:
You should not be appending this DOM tree directly to the page inside of this function, you should be returning the DOM tree and adding it to the DOM where the function was called.

Reply:
sounds good!